website/docs/docs/dbt-cloud-apis/apis-overview.md

```diff
@@ -12,7 +12,7 @@ Accounts on the _Team_ and _Enterprise_ plans can query the dbt Cloud APIs.
 
 dbt Cloud provides the following APIs:
 
-- The [dbt Cloud Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) can be used to administrate a dbt Cloud account.
+- The [dbt Cloud Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) can be used to administrate a dbt Cloud account. It can be called manually or via [the dbt Cloud Terraform provider](https://registry.terraform.io/providers/dbt-labs/dbtcloud/latest).
```

Comment:
```suggestion
- The [dbt Cloud Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) can be used to administrate a dbt Cloud account. It can be called manually or through [the dbt Cloud Terraform provider](https://registry.terraform.io/providers/dbt-labs/dbtcloud/latest).
```